server/sonar-server/src/test/java/org/sonar/server/qualityprofile/QProfileFactoryMediumTest.java

```diff
@@ -188,40 +191,48 @@ public void delete_descendants() {
 
   @Test
   public void do_not_delete_default_profile() {
-    db.qualityProfileDao().insert(dbSession, QProfileTesting.newXooP1("org-123"));
-    factory.setDefault(dbSession, XOO_P1_KEY);
+    QualityProfileDto profile = QualityProfileTesting.newQualityProfileDto()
+      .setDefault(true);
+    db.qualityProfileDao().insert(dbSession, profile);
```

Comment:
have you considered `db.qualityProfiles().insertQualityProfiles`?

it will save you from calling `dbSession.commit()` and I don't think `dbSession.clearCache()` is actually useful to anything (not used in UTs with DBTester but those on QP and Rules, I wonder whether it's just useless legacy code)